monai/networks/nets/quicknat.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from typing import Sequence, Tuple, Union
+
+import torch
+import torch.nn as nn
+from squeeze_and_excitation import squeeze_and_excitation as se1
+
+from monai.networks.blocks import Bottleneck, ConvConcatDenseBlock, Decoder, Encoder
+from monai.networks.blocks import squeeze_and_excitation as se
+from monai.networks.blocks.convolutions import ClassifierBlock
+from monai.networks.layers.factories import Act, Norm
+from monai.networks.layers.simplelayers import SkipConnectionWithIdx
+from monai.networks.layers.utils import get_dropout_layer, get_pool_layer
+from monai.utils import export
+
+__all__ = ["Quicknat"]
+
+
+@export("monai.networks.nets")
```

Comment:
This isn't used anymore actually so you can remove it.

Reply:
Ok! Thank you

Reply:
@ericspod Thank you for all the comments